sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -546,10 +546,15 @@ public virtual ResourceGroupCreateOrUpdateOperation StartSetTags(IDictionary<str
 
             try
             {
-                var resource = GetResource(cancellationToken);
-                var patch = new ResourceGroupPatchable();
-                patch.Tags.ReplaceWith(tags);
-                var originalResponse = RestClient.Update(Id.Name, patch, cancellationToken);
+                var tagsOperations = GetTagsOperations();
+                tagsOperations.DeleteAtScope(cancellationToken);
+                Tags newTags = new Tags();
+                foreach (var item in tags)
```

Comment:
You can use ReplaceWith extension method

Reply:
Done